marketplace-elements.styl

```diff
@@ -243,6 +243,7 @@ for $i in (0..3) {
 */
 .mkt-prompt {
     position: relative;
+    top: 10px;
```

Comment:
Does this add the grey space above the form on mobile? Can you use padding instead?